src/reference/antora/modules/ROOT/pages/cloudevents-transform.adoc

```diff
@@ -0,0 +1,133 @@
+[[cloudevents-transformer]]
+
+= CloudEvent Transformer
+
+[[introduction]]
+== Introduction
+
+The CloudEvent transformer converts Spring Integration messages into CloudEvent compliant messages.
+This transformer provides  support for the CloudEvents specification v1.0 with configurable output format and defining attributes and extensions using `Expression`s.
+
+[[cloudevent-transformer-overview]]
+=== Overview
+
+The CloudEvent transformer (`ToCloudEventTransformer`) extends Spring Integration's `AbstractTransformer` to convert messages to CloudEvent format.
```

Comment:
No need to say that `AbstractTransformer` is a Spring Integration class.
We are already here.